pkg/docker/index.html

```diff
@@ -336,6 +338,45 @@ <h1 translatable="yes">Can't connect to Docker</h1>
     </div>
   </div>
 
+  <!-- STORAGE -->
+
+  <div id="storage" class="container-fluid" hidden>
+    <ol class="breadcrumb">
+      <li><a translatable="yes">Containers</a></li>
+      <li class="active">Storage</li>
+    </ol>
+
+    <div id="storage-unsupported" hidden>
+      <div class="blank-slate-pf">
+        <h1 id="storage-unsupported-message"></h1>
+      </div>
+    </div>
+
+    <div id="storage-details">
+      <div class="col-md-4">
+        <h3 translatable="yes">Overview</h3>
+        <div id="storage-overview"></div>
+        <h3>
```

Comment:
> I think the second <h3> is bogus here.

No, I was confused.  Fixed by using a `span` instead of the `div`.

@garret could you have a look at the HTML coding in general?  How would you implement your mockup in HTML/CSS?